tests/fv/__init__.py

```diff
@@ -0,0 +1,84 @@
+import os
+import sh
+from sh import docker
+
+
+def setup_package():
+    """
+    Sets up docker images and host containers for running the STs.
+    """
+    pwd = sh.pwd().stdout.rstrip()
+
+    docker_exec = docker.bake("exec")
+    host1_exec = docker_exec.bake("-t", "host1", "bash", "-c")
+    host2_exec = docker_exec.bake("-t", "host2", "bash", "-c")
+
+    # We *must* remove all inner containers and images before removing the outer
+    # container. Otherwise the inner images will stick around and fill disk.
+    # https://github.com/jpetazzo/dind#important-warning-about-disk-usage
+    try:
+        host1_exec('docker rm -f $(docker ps -qa) ; docker rmi $(docker images -qa)')
+    except sh.ErrorReturnCode_1:
+        pass
+    try:
+        host2_exec('docker rm -f $(docker ps -qa) ; docker rmi $(docker images -qa)')
+    except sh.ErrorReturnCode_1:
+        pass
+    print "Containers and images within host containers removed."
+
+    try:
+        sh.docker.rm("-f", "host1", "host2")
+    except sh.ErrorReturnCode:
+        pass
+    print "Host containers removed."
+
+    # Pull and save each image, so we can use them inside the host containers.
+    print sh.bash("./build_node.sh").stdout
+    docker.save("--output", "calico-node.tar", "calico/node")
+    if not os.path.isfile("busybox.tar"):
```

Comment:
Would it make sense to save these tars to /tmp/calico-build is something similar?

This would avoid passing them into the build context when building the node or build containers.